apps/studio/components/interfaces/Settings/Database/DiskSizeConfiguration.tsx

```diff
@@ -86,66 +81,98 @@ const DiskSizeConfiguration = ({ disabled = false }: DiskSizeConfigurationProps)
       .max(Number(maxDiskSize), 'Must not be more than 200 GB'),
   })
 
+  const { data } = useDatabaseSizeQuery({
+    projectRef: project?.ref,
+    connectionString: project?.connectionString,
+  })
+  const databaseSizeBytesUsed = data?.result[0].db_size ?? 0
+
   return (
     <div id="diskManagement">
       <FormHeader title="Disk management" />
       {projectSubscriptionData?.usage_billing_enabled === true ? (
         <div className="flex flex-col gap-3">
           <Panel className="!m-0">
             <Panel.Content>
-              <div className="grid grid-cols-1 items-center lg:grid-cols-3">
-                <div className="col-span-2 space-y-1">
+              <div className=" ">
+                <div className="">
```

Comment:
Not sure why but I find this upsetting
```suggestion
              <div>
                <div>
```

Reply:
hah! omg that's the worst! sorry! totally upsetting.